code/modules/antagonists/traitor/objectives/demoralise_poster.dm

```diff
@@ -0,0 +1,161 @@
+/datum/traitor_objective/demoralise/poster
+	name = "Sow doubt among the crew %VIEWS% times using Syndicate propaganda."
+	description = "Use the button below to materialize a pack of posters, \
+		which will demoralise nearby crew members (especially those in positions of authority). \
+		If your posters are destroyed before they are sufficiently upset, this objective will fail. \
+		Try hiding some broken glass behind your poster before you hang it to give  \
+		do-gooders who try to take it down a hard time!"
+
+	/// Have we handed out a box of stuff yet?
+	var/granted_posters = FALSE
+	/// All of the posters the traitor gets, if this list is empty they've failed
+	var/list/posters = list()
```

Comment:
type this as the thing it contains, also uh, why do these need to be hard references?